packages/orchestration/test/staking-ops.test.ts

```diff
@@ -92,7 +99,7 @@ const makeScenario = () => {
 
       '/cosmos.staking.v1beta1.MsgBeginRedelegate': _m => {
         const response = MsgBeginRedelegateResponse.fromPartial({
-          completionTime: new Date('2025-12-17T03:24:00Z'),
+          completionTime: dateToTimestamp(new Date('2025-12-17T03:24:00Z')),
```

Comment:
here's hoping we manage to completely replace the mock stuff here with using ibc-mocks

Reply:
Agree. It seems there's parity here with `stake-ica.test.ts` minus the wallet offers.

Will +1 that we should be able to use a macro and test LocalOrchestrationAccount in the same flow. Currently there are a two small blockers to that:
  - LOA `delegate` and `undelegate` should accept a ChainAddress obj instead of a string
  - LOA `undelegate` should accept an array of delegations